snippets/remote_execution_ssh_keys.erb

```diff
@@ -32,4 +32,7 @@ EOF
 chmod 700 <%= ssh_path %>
 chmod 600 <%= ssh_path %>/authorized_keys
 chown -R <%= "#{ssh_user}:" %> <%= ssh_path %>
+
+# Restore SELinux context if a linux system with restorecon:
+command -v restorecon && restorecon -RvF <%= ssh_path %>
```

Comment:
as this snippet is running on quite some OSes, could there be some check added before that block? I'm getting exit code 1 on this on e.g. Debian.